intelmq/bots/experts/filter/expert.py

```diff
@@ -82,21 +82,21 @@ def process(self):
             except ValueError:
                 self.logger.error("Could not parse time.source %s.", event.get('time.source'))
             else:
-                if type(self.not_after) is datetime and event_time > self.not_after:
+                if isinstance(self.not_after, datetime) and event_time > self.not_after:
                     self.acknowledge_message()
                     self.logger.debug("Filtered out event with time.source %s.", event.get('time.source'))
                     return
-                if type(self.not_before) is datetime and event_time < self.not_before:
+                if isinstance(self,not_before, datetime)  and event_time < self.not_before:
```

Comment:
typo: ,
and added whitespace
```suggestion
                if isinstance(self.not_before, datetime) and event_time < self.not_before:
```